index.html

```diff
@@ -163,13 +163,13 @@ <h3>Pratyasha Kalita <br><span class="position">Artist</span></h3>
                     </a>
                   </div>
 
-                </div>
+                </div>`
```

Comment:
can you just correct this, I guess it was added by mistake.

Reply:
on it